packages/flipt-sdk/types/createFliptSDK.d.ts

```diff
@@ -1,16 +1,21 @@
-import type BatchEvalutationResponse from './@types/BatchEvalutationResponse';
+import type BatchEvaluationResponse from './@types/BatchEvaluationResponse';
 import type Context from './@types/Context';
-import type Evalutation from './@types/Evaluation';
+import type EvaluationResponse from './@types/EvaluationResponse';
```

Comment:
```suggestion
import type { 
   EvaluationResponse,  
   BatchEvaluationResponse, 
   Context,
   FliptConfig
} from './@types';
```

Não seria uma boa importar os tipos de um Baú, seguindo nosso padrão?

Reply:
Importar do @mechamobau? Acho que era pra ser barril hsaushaushua

Reply:
kakakakakakakakakka Barril* tava pensando no que teria na palta do @mechamobau pra guilda kkkkkkkkk